create_pnor_image.pl

```diff
@@ -131,6 +131,7 @@
 $build_pnor_command .= " --binFile_SECBOOT $scratch_dir/secboot.bin.ecc";
 $build_pnor_command .= " --binFile_VERSION $openpower_version_filename";
 $build_pnor_command .= " --binFile_IMA_CATALOG $scratch_dir/ima_catalog.bin.ecc";
+$build_pnor_command .= " --binFile_WOFDATA $scratch_dir/wofdata.bin.ecc";
```

Comment:
My mistake, forgot that we didn't branch this repo like the others.

The contents will be coming from a file (or possibly set of files) out of the system mrw xml repos.  The data doesn't exist yet, but I wanted to pre-seed the layouts to make bringup simpler.

Why would the 'auto update' be broken?  I am currently just putting blank data in there.  That should be fine for now.